wrappers/ios/ci/build.sh

```diff
@@ -405,7 +368,7 @@ abspath() {
 
 # Setup environment
 setup
- 
+
 # Build 3rd party libraries
 build_crypto
```

Comment:
Is this step still necessary? When is the iOS build of openssl used? Does it replace openssl installed via brew?

Reply:
This step is required by `build_libvcx`, because `libvcx/build.rs` requires variable `OPENSSL_LIB_DIR` to be passed.